kitten_options_utils.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Sequence, Tuple, Iterable
+
+from kitty.conf.utils import KittensKeyDefinition, key_func, parse_kittens_key
+from kitty.rgb import Color, parse_sharp
+
+func_with_args, args_funcs = key_func()
+FuncArgsType = Tuple[str, Sequence[Any]]
+
+
+def color_parser(val: str) -> Color:
+    assert val.startswith('#'), f"Color should start with #, got {val}"
+    color = parse_sharp(val[1:])
+    if color is None:
+        raise ValueError(f"Couldn't parse {val} as a color")
+    return color
+
+
+def parse_map(val: str) -> Iterable[KittensKeyDefinition]:
+    x = parse_kittens_key(val, args_funcs)
+    if x is not None:
+        yield x
+
+
+# TODO what are these functions supposed to do?
```

Comment:
Same thing [they were doing before][before]: take a string of arguments, split it as relevant to the command, validate each argument, and return a value or tuple.

[before]: https://github.com/yurikhan/kitty_grab/pull/14/files#diff-8087c975402900536bc73314799fb77e1dd7f11e5bc24ddda2f69e8f571a9db3L481-L493

Reply:
Oh I see, fixed in 9d377491f7500f92ca0a3fb2c93b9d0d461dd9b3